eras/conway/impl/src/Cardano/Ledger/Conway/Translation.hs

```
@@ -127,15 +128,25 @@ instance Crypto c => TranslateEra (ConwayEra c) EpochState where
         }
 
 instance Crypto c => TranslateEra (ConwayEra c) DState where
-  translateEra _ DState {..} = pure DState {..}
+  translateEra ConwayGenesis {..} DState {..} =
+    pure
+      DState
+        { dsUnified = UMap.unUView $ Map.foldrWithKey' UMap.insert' (UMap.DRepUView dsUnified) cgDRepDelegs
+        , ..
+        }
```

Nooooo!!!! This is not the way it is suppose to be done. Pleas revert it back

This would allow for uncontrolled insertion of drep delegations on mainnet

Please, study on how it is done with stake pools with ShelleyGenesis. 

We need a new `EraTransition era => ConwayEraTransition era` type class that will have two new lenses and separate function that uses them in order to implement injection of DReps and delegations into the ledger state. Those lenses also must fail hard for `Mainnet`

See `Cardano.Ledger.Shelley.Transition` for examples on how to get it done.